src/react/components/ui/dialog.tsx

```
@@ -0,0 +1,142 @@
+import * as DialogPrimitive from '@radix-ui/react-dialog'
+import { XIcon } from 'lucide-react'
+import * as React from 'react'
+
+import { cn } from '@/client/lib/utils/cn'
+
+
+function Dialog({
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)